src/database/utils/cleanEmoji.js

```diff
@@ -4,6 +4,10 @@ export function cleanEmoji (emoji) {
   if (!emoji) {
     return emoji
   }
+  // if inside a Firefox content script, need to clone the emoji object to prevent Firefox from complaining about cross-origin object
+  if (typeof wrappedJSObject !== 'undefined') {
+      emoji = structuredClone(emoji)
```

Comment:
```suggestion
    emoji = structuredClone(emoji)
```

StandardJS is complaining about whitespace.